### PR TITLE
change fat arrow functions

### PR DESCRIPTION
**Pull request by dustinvforsyth**
fix for IE bug where input is not being displayed

### CODE REVIEW

File: titanium-address-input.html [hunk 1]
```diff
@@ -159,13 +159,32 @@
...
-						let stateComponent = place.address_components.filter(o => o.types.some(p => p ===
-							'administrative_area_level_1'))[0];
-						let streetNumberComponent = place.address_components.filter(o => o.types.some(p => p === 'street_number'))[
-							0];
-						let streetAddressComponent = place.address_components.filter(o => o.types.some(p => p === 'route'))[0];
-						let cityComponent = place.address_components.filter(o => o.types.some(p => p === 'locality'))[0];
-						let zipCodeComponent = place.address_components.filter(o => o.types.some(p => p === 'postal_code'))[0];
+						let stateComponent = place.address_components.filter(function (o) {
+							o.types.some(function (p) {
+								p ===
```
Comment (gfportfolio) [inline titanium-address-input.html]: formatting looks wrong here.

**Review by kaseyhinton — APPROVED** [review]
I am okay with this once the formatting is fixed.

**Review by AwsomeWyatt — APPROVED** [review]

**Review by ShawnJohnson14 — APPROVED** [review]

**Review by antwonthegreat — APPROVED** [review]

**Comment by aarondrabeck** [comment]
How about the Polymer 2 fat arrow functions?

**Comment by dustinvforsyth** [comment]
Shoot ya let me get those ones I will add another pull request

**Pull request merged**